library/ui-styles/src/main/res/values/dimens.xml

```diff
@@ -18,8 +18,8 @@
 
     <item name="dialog_width_ratio" format="float" type="dimen">0.75</item>
 
-    <dimen name="pill_avatar_size">16dp</dimen>
-    <dimen name="pill_min_height">20dp</dimen>
+    <dimen name="pill_avatar_size">20dp</dimen>
```

Comment:
Should the avatar size be in `sp` units so that it scales with the text size?

Reply:
Good remark. I will test it.